clients/gateway.go

```diff
@@ -17,7 +17,7 @@ type GatewayClient struct {
 
 func NewGatewayClient(baseUrl string) *GatewayClient {
 	return &GatewayClient{
-		baseUrl: baseUrl,
+		baseUrl: baseUrl + "/feeder_gateway/",
```

Comment:
It would be good to extract it into `const`

Reply:
done